editor/src/clj/editor/model_scene.clj

```diff
@@ -355,8 +352,8 @@
      :renderable {:render-fn render-outline
                   :tags #{:model :outline}
                   :batch-key nil ; Batching is disabled in the editor for simplicity.
-                  :select-batch-key model-scene-resource-node-id
-                  :passes [pass/outline]}
+                  :select-batch-key :not-rendered ; The render-fn only does anything during the outline pass.
```

Comment:
A bit confusing, at first I thought this is a specially recognized key, but it's not :)